components/Image/image.html

```diff
@@ -0,0 +1,179 @@
+              <li><a href="" class="group-item">Button</a></li>
+              <li><a href="" class="group-item">Card</a></li>
+              <li><a href="" class="group-item">Image</a></li>
+              <li><a href="" class="group-item">Input</a></li>
+              <li><a href="" class="group-item">List</a></li>
+            </ul>
+          </div>
+        </aside>
+        <div class="docs-container">
+          <div class="docs-content">
+            <h1 class="docs-heading">Image</h1>
+            <p class="docs-primary-content">
+                Documentation and examples for opting images into responsive behavior (so they never become larger than their parent elements) and add lightweight styles to them—all via classes.
+            </p>
+            <div class="divider"></div>
+            <h2 class="docs-sub-heading">Responsive Images</h2>
+            <p class="docs-secondary-content">
+                Images can be made responsive using the <code class="markdown-code">.img-fluid</code> class
+            </p>
+            <div class="display-container flex justify-center">
+              <img class="img-fluid" src="https://images.unsplash.com/photo-1500964757637-c85e8a162699?ixlib=rb-1.2.1&ixid=MnwxMjA3fDB8MHxzZWFyY2h8MTV8fGNvbG9yZnVsJTIwYWJzdHJhY3R8ZW58MHwwfDB8fA%3D%3D&auto=format&fit=crop&w=500&q=60" alt="">
+            </div>
+            <div class="code-container">
+              <pre>
+                      <code class="language-html">
+        &lt;img class="img-fluid" src="img-url" alt=""&gt;
+                      </code>
+                  </pre>
+            </div>
+            <div class="divider"></div>
+            <h2 class="docs-sub-heading">Rounded Images</h2>
+            <p class="docs-secondary-content">
+                Adjust roundedness of images using the border-radius utility classes.
+            </p>
+            <div class="display-container flex justify-center wrap gap-20">
+              <img class="img-fluid rounded-lg" src="https://images.unsplash.com/photo-1500964757637-c85e8a162699?ixlib=rb-1.2.1&ixid=MnwxMjA3fDB8MHxzZWFyY2h8MTV8fGNvbG9yZnVsJTIwYWJzdHJhY3R8ZW58MHwwfDB8fA%3D%3D&auto=format&fit=crop&w=500&q=60" alt="">
+              <img class="img-fluid rounded-md" src="https://images.unsplash.com/photo-1500964757637-c85e8a162699?ixlib=rb-1.2.1&ixid=MnwxMjA3fDB8MHxzZWFyY2h8MTV8fGNvbG9yZnVsJTIwYWJzdHJhY3R8ZW58MHwwfDB8fA%3D%3D&auto=format&fit=crop&w=500&q=60" alt="">
+              <img class="img-fluid rounded-sm" src="https://images.unsplash.com/photo-1500964757637-c85e8a162699?ixlib=rb-1.2.1&ixid=MnwxMjA3fDB8MHxzZWFyY2h8MTV8fGNvbG9yZnVsJTIwYWJzdHJhY3R8ZW58MHwwfDB8fA%3D%3D&auto=format&fit=crop&w=500&q=60" alt="">
+              <img class="img-fluid rounded-xs" src="https://images.unsplash.com/photo-1500964757637-c85e8a162699?ixlib=rb-1.2.1&ixid=MnwxMjA3fDB8MHxzZWFyY2h8MTV8fGNvbG9yZnVsJTIwYWJzdHJhY3R8ZW58MHwwfDB8fA%3D%3D&auto=format&fit=crop&w=500&q=60" alt="">
+            </div>
```

Comment:
Always give alt (alternate) in (avatar, badge).

Reply:
On it!